Mathlib/Tactic/ToAdditive.lean

```diff
@@ -435,6 +435,44 @@ def isInternal' (declName : Name) : Bool :=
   | .str _ s => "match_".isPrefixOf s || "proof_".isPrefixOf s || "eq_".isPrefixOf s
   | _        => true
 
+/-- Find the target name of `pre` and all created auxiliary declarations. -/
+def findTargetName (env : Environment) (src pre tgt_pre : Name) : CoreM Name :=
+  let privatePrefix := privateHeader ++ env.mainModule |>.num 0
```

Comment:
Is there a reason you're not using `privateToUserName?` and `mkPrivateName` here?  I'm not excited about hardcoding the current private name encoding.

Reply:
Ah, I didn't see `privateToUserName?` in the private file.